### PR TITLE
SAP S/4HANA OData Cloud Guide 1.2.0 M4

### PR DESCRIPTION
**Pull request by xBATx**
Update user manual for release 1.2.0 of sap-s4hana.

### CODE REVIEW

File: modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc [hunk 1]
```diff
@@ -1,35 +1,29 @@
...
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../../assets/images/
 
-Support Category: Premium
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
 
-SAP S/4HANA Cloud Connector v1.1
+SAP S/4HANA Cloud Connector v1.2
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc]: Add Odata?
```suggestion
SAP S/4HANA Odata Cloud Connector v1.2
```

File: modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc [hunk 2]
```diff
@@ -1,35 +1,29 @@
...
 
-The Anypoint Connector for SAP S/4HANA Cloud provides full support for 
-the query, get, create, update, delete entities, and execute functions using the ODATA API exposed by SAP S/4HANA.
+Anypoint Connector for SAP S/4HANA OData Cloud (S/4HANA OData Cloud Connector) provides full support for 
+the query, get, create, update, delete entities, and execute functions using the OData API exposed by SAP S/4HANA.
 
-This connector lets you perform operations to:
+The connector lets you perform operations to:
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc]: lets, allows > enables
```suggestion
The connector enables you to perform operations to:
```

File: modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc [hunk 3]
```diff
@@ -1,35 +1,29 @@
...
 
-* Create, update, and delete entities.
-* Retrieve a single entity or query multiple entities.
-* Invoke a remote function.
+* Create, update, and delete entities
+* Retrieve a single entity or query multiple entities
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc]: Do you suppose that this should be two list items, since both retrieve and query, and single and multiple, are different?

File: modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc [hunk 4]
```diff
@@ -1,35 +1,29 @@
...
 
-Support Category: Premium
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
 
-SAP S/4HANA Cloud Connector v1.1
+SAP S/4HANA Cloud Connector v1.2
 
-The Anypoint Connector for SAP S/4HANA Cloud provides full support for 
-the query, get, create, update, delete entities, and execute functions using the ODATA API exposed by SAP S/4HANA.
+Anypoint Connector for SAP S/4HANA OData Cloud (S/4HANA OData Cloud Connector) provides full support for
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc]: Should we add "SAP" to the abbreviated name as well?
```suggestion
Anypoint Connector for SAP S/4HANA OData Cloud (SAP S/4HANA OData Cloud Connector) provides full support for 
```

File: modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc [hunk 5]
```diff
@@ -1,35 +1,29 @@
...
-To use this connector you need the following:
-
-* A developer account or other valid SAP S/4HANA instance.
-* A MuleSoft Enterprise License.
-
-For software requirements and compatibility
-information, see the xref:release-notes::connector/sap-s4-hana-connector-release-notes-mule-4.adoc[SAP S4/HANA Connector Release Notes].
-
-To use this connector with Maven, view the `pom.xml` dependency information in the *Dependency Snippets* in Anypoint Exchange.
+* Familiarity with S/4HANA OData, Mule, Anypoint Connectors, and the Anypoint Studio essentials.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc]: Add SAP? and other stuff:
```suggestion
* Familiarity with SAP S/4HANA OData, Mule runtime engine (Mule), Anypoint Connectors, and Anypoint Studio essentials
```

File: modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc [hunk 6]
```diff
@@ -60,9 +54,9 @@ image::sap/sap-hana-http-listener.png[]
...
 |Protocol |Protocol selected for the HTTP endpoint. This can be HTTP or HTTPS.
-|Host |IP address on which your Mule app listens for requests.
-|Port |Port address on which your Mule app listens for requests.
-|Base Path |Path on which your Mule app listens for requests.
+|Host |IP address at which the Mule app listens for requests.
+|Port |Port address at which the Mule app listens for requests.
+|Base Path |Path at which the Mule app listens for requests.
 |===
 +
 . Select the plus sign to add a component.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc]:
```suggestion
. Select the plus sign (+) to add a component.
```

File: modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc [hunk 7]
```diff
@@ -147,6 +156,8 @@ image::sap/sap-hana-conf.png[]
 [%header,cols="30s,70a"]
 |===
 |Studio Field |Description
+|Discovery Service Enabled |Feature switch for service discovery. If disabled, Service discovery relative path is not applied.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc]:
```suggestion
|Discovery Service Enabled |Feature switch for service discovery. If disabled, the service discovery relative path is not applied.
```

File: modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc [hunk 8]
```diff
@@ -155,12 +166,12 @@ image::sap/sap-hana-conf.png[]
 |Proxy configuration |Configuration for executing requests through a proxy.
 |===
 +
-. Click *OK* to save the global connector configuration.
+. Click OK to save the global connector configuration.
```
Comment (kbostrom) [inline modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc]: Why did you change the tagging for UI control label?

**Review by kbostrom — APPROVED** [review]
Done!

**Review by george-hoffer — APPROVED** [review]

**Review by george-hoffer — APPROVED** [review]
